gitlabform/lists/projects.py

```diff
@@ -130,10 +133,31 @@ def _verify_if_projects_exist_and_get_archived_projects(
                 if project_object["archived"]:
                     archived.append(project_object["path_with_namespace"])
             except NotFoundException:
-                fatal(
-                    f"Configuration contains project {project} but it cannot be found in GitLab!",
-                    exit_code=EXIT_INVALID_INPUT,
+                debug("Could not find '%s'", project)
+                debug(
+                    "Checking if it's a project that needs to be transferred from elsewhere"
```

Comment:
Do you know why we need to do this here? Seems like repeat logic that was already done but maybe I'm missing something.

Reply:
I moved the second debug message to the _get_project_transfer_source_from_config method, so that it's not duplicated in 2 places. We do need the `except NotFoundException` here since this method is checking if a project exists which it won't for projects being transferred.

Reply:
Okay, let's close this thread. If needed, will open new discussion from latest code.